doc/news/_preparation_next_release.md

```diff
@@ -175,6 +175,9 @@ Many problems were resolved with the following fixes:
 - The [Shell Recorder][] counts the number of executed tests properly again.
 - CMake now fails if the required plugins [list](http://libelektra.org/plugins/list) or [spec](http://libelektra.org/plugins/spec) (on
    non-[MinGW](http://mingw.org) platforms) are missing from the current build configuration.
+- The [Lua](http://libelektra.org/plugins/lua), [Python 2](http://libelektra.org/plugins/python2),
```

Comment:
this does not mention you, nor the travis nor the change with the exception handler

Reply:
> this does not mention you…

This is intentional. I do not like adding text that basically says “thanks to me”. The main update here – adding `check_binding_included` – was done by Thomas (Wahringer) anyway.

> …, nor the travis [change]…

I do not think changing a compiler version for one build job is interesting.

> nor the change with the exception handler

Same as above.